### ISSUE TITLE
Issue with demo_parse_css function

### ISSUE THREAD

**Issue by BassemN** [valendesigns/option-tree-theme#11]
There is a bug on [demo_get_option( 'demo_css', true )](https://github.com/valendesigns/option-tree-theme/blob/master/index.php#L239) specifically with the typography option type as it doesn't parse Google fonts and just parse default fonts

**Comment by BassemN** [comment]
If I select a Google font, it returns `no value`

**Comment by valendesigns** [comment]
Yeah, the demo is likely missing a Google Fonts filter, so its attempts to display CSS like OptionTree saves to `dynamic.css` fail. But it's an issue only in the theme. I'll fix it after WordPress 4.2 releases, as I'm busy working on Core tickets & testing right now. Thanks for posting the bug report though.
